### ISSUE TITLE
Trouble installing quicksect package

### ISSUE THREAD

**Issue by arjungoyal-codex** [medspacy/medspacy#87]
I'm attempting to download medspacy using pip install in a Conda environment. When I try `nlp = medspacy.load()`, I get an error regarding the quicksect package which is imported sequentially by PyRuSHSentencizer. I've checked the quicksect package [documentation](https://github.com/brentp/quicksect/blob/master/src/quicksect.pyx), and I can't seem to find a definition for quicksect1. 

<img width="822" alt="Screen Shot 2021-09-08 at 4 23 59 PM" src="https://user-images.githubusercontent.com/81549595/132598328-5c95d3df-3e79-4bd3-bc96-1883a83cce26.png">

This are the requirements that pip install medspacy downloaded: 
PyRuSH-1.0.3.5
jsonschema-3.2.0 
medspacy-0.1.0.2 
medspacy-quickumls-2.4.1 
spacy-2.3.2

I've also tried installing quicksect separately and that fails too with 'tp_print' instead of 'tp_dict' errors that I'm not sure how to interpret.

<img width="1006" alt="Screen Shot 2021-09-08 at 4 31 59 PM" src="https://user-images.githubusercontent.com/81549595/132598938-957bd52b-4d28-4d65-b91b-f19a920a7f83.png">

Any help would be appreciated! Thank you!

**Comment by farbodr** [comment]
I'm having a similar issue. Any suggestions?

**Comment by abchapman93** [comment]
It looks like you're using Python 3.9. Could you try using Python 3.8 and see if that fixes it? Python 3.9 isn't currently supported with medspaCy and a number of its dependencies. If that's not the issue, I'll ask @jianlins  PyRuSH developer if he has any suggestions.

**Comment by jsilvestri** [comment]
The problem is that a file for quicksect is missing. You can get around this for Python 3.9 as follows:


# 1. manually download the current version of [quicksectx](https://pypi.org/project/quicksectx/) and unzip it.  
```
curl -o quicksectx.tar.gz https://files.pythonhosted.org/packages/bb/ed/88af405793bc56aa756447450f16ff095d4ccd38cc3aa3ac1afff3983a05/quicksectx-0.3.0.tar.gz#sha256=a419f376574b1ebf6f77233d31decef7ffc89daaeda9b5da6d6b2b7ad72adddf
tar xvzf quicksectx.tar.gz
```

# 2. manually download [quicksect/quicksect.pyx](https://raw.githubusercontent.com/jianlins/quicksectx/master/quicksect/quicksect.pyx) 
put this file put it into the directory quicksectx-0.3.0/quicksect/quicksect.pyx

# 3. from the directory that contains the quicksectx-0.3.0 folder, run:
```
pip install quicksectx-0.3.0/
```

This should work!  Please note that I have opened an [issue](https://github.com/jianlins/quicksectx/issues/3) to try to get this fixed.

**Comment by jsilvestri** [comment]
This bug in quicksectx has now been fixed in version 0.3.1. See https://github.com/jianlins/quicksectx/issues/3

@arjungoyal-codex, @farbodr, @abchapman93 I believe this issue can be closed out.

**Comment by burgersmoke** [comment]
I believe that between medspacy and quicksectx, there are now multiple workarounds.

Medspacy 1.0.0 (released yesterday) now supports Python 3.9.  So that is one option.  The other is to use quicksect 0.3.1 as @jsilvestri kindly mentions above.

As such, I am inclined to Close this ticket, but please open a new one is there are still issues we need to resolve.
